Pipfile

```diff
@@ -0,0 +1,13 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[requires]
+python_version = "3.9"
```

Comment:
Do we want to avoid the `networkx` issues? If so, we could pin this to 3.8.

Reply:
I'm sort of ok with this except that I have not found it's always true that there are networkx issues if you use 3.9. Can you give more details re when that problem arises?